### ISSUE TITLE
Add book field for ISBN

### ISSUE THREAD

**Issue by nosegraze** [nosegraze/book-database#64]

**Comment by nosegraze** [comment]
Link review with edition maybe?

**Comment by GuillouTran** [comment]
Have you by chance able to do this?

**Comment by ashleyfae** [comment]
@GuillouTran It's available when adding an owned edition, but not as part of the main book data.

**Comment by GuillouTran** [comment]
@ashleyfae hmm. That’s great. I was looking for something like this. How do you host this? Cause it’s pure php right?

**Comment by ashleyfae** [comment]
@GuillouTran  Yes it's pure PHP. Though this is a plugin for WordPress so it also requires that WordPress be installed. You'll need to look up instructions on how to install WordPress first.

**Comment by GuillouTran** [comment]
Ah great! So, I need to install wordpress(it's no problem!), but may I ask if you by chance have a demo?

**Comment by ashleyfae** [comment]
Sorry I don't have a demo.

**Comment by GuillouTran** [comment]
Then maybe a screenshot or some kind of demonstration(I just want to make sure I won’t waste my time :)

**Comment by ashleyfae** [comment]
@GuillouTran There are some screenshots available here: https://shop.nosegraze.com/product/book-database/

**Comment by GuillouTran** [comment]
I see that it's for sale. Wonder why it's on GitHub?

**Comment by ashleyfae** [comment]
Just because :shrug:

**Comment by GuillouTran** [comment]
lol
